bin/i18n/sync-down.rb

```diff
@@ -5,39 +5,39 @@
 # https://crowdin.com/project/codeorg
 
 require_relative 'i18n_script_utils'
-require 'open3'
+
+require 'cdo/crowdin/utils'
+require 'cdo/crowdin/project'
 
 def sync_down
   I18nScriptUtils.with_syncronous_stdout do
```

Comment:
Not relevant to _this_ PR: Has anyone pointed out that "synchronous" is misspelled?

Reply:
Lol, no :upside_down_face: 

I'll fix in a separate PR!